src/App.tsx

```diff
@@ -3,24 +3,43 @@ import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { NavigationContainer } from '@react-navigation/native';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
-import { StyleSheet, Text, View } from 'react-native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { StyleSheet, View, Text } from 'react-native';
 import Ionicons from '@expo/vector-icons/Ionicons';
 
 import Contributors from './Contributors';
-import ButtonComponent from './components/ButtonComponent';
+import WelcomeScreen from './screens/WelcomeScreen';
+import LoginScreen from './screens/LoginScreen';
+import SignUpScreen from './screens/SignUpScreen';
 
 const Tab = createBottomTabNavigator();
+const Stack = createNativeStackNavigator();
 
-const HomeScreen = () => {
+const TestScreen = () => {
```

Comment:
Why did you change this name to TestScreen?

Reply:
Because now we have the Welcome Screen as the Home Screen. Last time I deleted the Home Screen. But this time I thought I will keep it as a Test Screen in case if we want to use it to test out any reusable component we create.

Reply:
If you want I can delete it.

Reply:
@sivaShilpa I don't think we need it :) if folks want to test something out, they can add it in their branch. Alternatively, it could be behind some env configuration so it's not visible all the time but we can create a separate task for that 👍🏽